src/CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/MessageQueue.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Queues;
+using Azure.Storage.Queues.Models;
+
+namespace CoreWCF.Channels
+{
+    internal class MessageQueue : IQueueBase
+    {
```

Comment:
I'm not seeing this class used anywhere

Reply:
updated code now. I hadnt updated the code to use this instead of AzureQueue directly.